website/home/app/page.tsx

```diff
@@ -55,110 +56,61 @@ const WriteDraft = gensx.Component<WriteDraftProps, string>(
   },
 );
 `,
-    workflows: `import * as gensx from 'gensx';
-import { OpenAIProvider } from 'gensx/openai';
-import { Research, WriteDraft, EditDraft } from './writeBlog';
+    observability: (
+      <div className="relative">
+        <Image
+          src="/assets/home/trace.png"
+          alt="GenSX Workflow Tracing Dashboard"
+          width={1000}
+          height={600}
+          className="rounded-lg shadow-lg w-full border border-gray-200 dark:border-gray-800"
+        />
+        <div className="absolute top-0 left-0 w-full h-full bg-gradient-to-b from-transparent to-white/10 dark:to-black/10 rounded-lg pointer-events-none"></div>
+      </div>
+    ),
+    stateful: `import * as gensx from '@gensx/core';
```

Comment:
Personally, i think this would be more compelling if we just showed a simplified version of the chat memory example